beacon-chain/rpc/beacon/blocks.go

```diff
@@ -182,7 +182,7 @@ func (bs *Server) StreamBlocks(_ *ptypes.Empty, stream ethpb.BeaconChain_StreamB
 		select {
 		case event := <-blocksChannel:
 			if event.Type == blockfeed.ReceivedBlock {
-				data, ok := event.Data.(*blockfeed.ReceivedBlockData)
+				data, ok := event.Data.(blockfeed.ReceivedBlockData)
```

Comment:
Turns out we were sending the wrong type over this event feed, so the slasher would never get block events. This fixes the issue.